structuralcodes/core/_section_results.py

```diff
@@ -106,20 +106,9 @@ def __format__(self, spec: str) -> str:
         """Defines the format for returning the string representation.
 
         Arguments:
-            spec (str): The string specifying the format. Special case
-            "cracked" instead of gross may be defined.
+            spec (str): The string specifying the format.
         """
-        # output_string = 'Gross Concrete Section Properties:\n'
-        if 'cracked' in spec.lower():
-            type = 'Cracked'
-            spec = spec.replace(
-                'cracked', ''
-            ).strip()  # remove "cracked" from spec
-        else:
-            type = 'Gross'
-
-        output_string = f'{type} Concrete Section Properties:\n'
-
+        output_string = 'Concrete Section Properties:\n'
```

Comment:
I missed this from my previous contributions, but should the title should be "Section Properties" probably?

Reply:
Fixed in c453e63.